actions/views/root.js

```diff
@@ -48,20 +48,19 @@ export function unregisterPluginTranslationsSource(pluginId) {
 }
 
 export function loadTranslations(locale, url) {
-    return (dispatch) => {
+    return async (dispatch) => {
         const translations = {};
         Object.values(pluginTranslationSources).forEach((pluginFunc) => {
             Object.assign(translations, pluginFunc(locale));
         });
+        let localeTranslations = en;
+
+        // Need to go to the server for languages other than English
 
-        // No need to go to the server for EN
-        if (locale === 'en') {
-            copyAndDispatchTranslations(dispatch, translations, en, locale);
-            return;
+        if (locale !== 'en') {
+            localeTranslations = await Client4.getTranslations(url);
         }
-        Client4.getTranslations(url).then((serverTranslations) => {
-            copyAndDispatchTranslations(dispatch, translations, serverTranslations, locale);
-        }).catch(() => {}); // eslint-disable-line no-empty-function
+        copyAndDispatchTranslations(dispatch, translations, localeTranslations, locale);
```

Comment:
Since we're dispatching this action once, lets remove `copyAndDispatchTranslations ` and just dispatch inline
```
dispatch({
        type: ActionTypes.RECEIVED_TRANSLATIONS,
        data: {
            locale,
            translations,
        },
    });
```